pyengine/katana.py

```diff
@@ -32,14 +31,19 @@ def matcher(side,qty,book):
         bestid = 'none'
 
         for quote in book.keys():
-            print('trying quote {}'.format(quote))
             price = book[quote][0]
 
             # the first quote is inherently the best price as there is nothing to compare to
             if bestprice == 'none':
                 bestprice = price
 
             if compareprice(side,price,bestprice):
+
+                # log quotes that were evaluated but not chosen on a single line
+                if len(unusedquotes) > 0:
+                    print('following quotes were evaluated but not selected for this round: {}'.format(unusedquotes))
+                    unusedquotes = ''
+
                 print('{}:{} selected over {}:{}'.format(quote,price,bestid,bestprice))
```

Comment:
we should probably get rid of this line too. or change it to only say which quote was selected.